docs/es/week05/05-2.md

```diff
@@ -0,0 +1,340 @@
+<!-- 
+---
+lang-ref: ch.05-2
+title: Optimization Techniques II
+authors: Guido Petri, Haoyue Ping, Chinmay Singhal, Divya Juneja
+date: 29 February 2020
+typora-root-url: 05-3
+---
+-->
+---
+lang-ref: ch.05-2
+title: Técnicas de Optimización II
+authors: Guido Petri, Haoyue Ping, Chinmay Singhal, Divya Juneja
+date: 29 Febrero 2020
+typora-root-url: 05-3
+---
```

Comment:
This won't work.
There's no language info, translator, translation date.
Please, check other headers.
This part is processed by the program that builds the website.